src/main/java/org/interledger/InterledgerAddress.java

```diff
@@ -271,7 +275,7 @@ public int hashCode() {
 
       @Override
       public String toString() {
-        return "InterledgerAddress.Impl{value='" + value + "'}";
+        return this.getValue();
```

Comment:
Actually, while this is a trivial change, I'm going to move it to another PR - I feel icky sliding this into this PR without some discussion (even if it is a trivial thing).